packages/fiori/src/SideNavigationSelectableItemBase.ts

```diff
@@ -194,13 +201,31 @@ class SideNavigationSelectableItemBase extends SideNavigationItemBase {
 	}
 
 	_onkeydown(e: KeyboardEvent) {
-		if (isSpace(e)) {
+		const isRTL = this.effectiveDir === "rtl";
+
+		if (isSpace(e) || isRight(e) || isLeft(e)) {
 			e.preventDefault();
 		}
 
 		if (isEnter(e)) {
 			this._activate(e);
 		}
+
+		if (isMinus(e) && !this.sideNavCollapsed) {
+			this._activate(e);
+		}
+
+		if (isPlus(e) && this.sideNavCollapsed) {
+			this._activate(e);
+		}
+
+		if ((isRTL ? isLeft(e) : isRight(e)) && this.sideNavCollapsed) {
+			this._activate(e);
+		}
+
+		if ((isRTL ? isRight(e) : isLeft(e)) && !this.sideNavCollapsed) {
```

Comment:
This will select the item, as well of closing the popup, but the item shouldn't be selected. You can modify this to:

```
if ((isRTL ? isRight(e) : isLeft(e)) && this.inPopover) {
			this.associatedItem?.sideNavigation?.closePicker();
		}
```
You will need to define the `associatedItem` as a property:

`associatedItem?: SideNavigationItemBase;`